aten/src/ATen/native/Embedding.cpp

```diff
@@ -53,18 +53,21 @@ Tensor embedding(const Tensor & weight, const Tensor & indices,
 }
 
 Tensor embedding_backward_symint(
-    const Tensor & grad, const Tensor & indices, SymInt num_weights,
-    int64_t padding_idx, bool scale_grad_by_freq, bool sparse) {
+    const Tensor & grad, const Tensor & indices, c10::SymInt num_weights,
+    c10::SymInt padding_idx, bool scale_grad_by_freq, bool sparse) {
   if (sparse) {
     // TODO: if we teach sparse tensor how to propagate symints, the guard
     // here is not strictly necessary.  However, we think it is fine as is
     // because num weights is derived from a parameter and therefore
     // typically not varying.
     return at::embedding_sparse_backward(
-    grad, indices, num_weights.guard_int(__FILE__, __LINE__), padding_idx, scale_grad_by_freq);
+      grad, indices,
+      num_weights.guard_int(__FILE__, __LINE__),
+      padding_idx.guard_int(__FILE__, __LINE__),
```

Comment:
Heads up: is this okay? I've just copied what's done to `num_weights`.

Reply:
It is fine pending symintifying this function